libbeat/scripts/Makefile

```diff
@@ -137,8 +139,10 @@ prepare-tests:
 	mkdir -p ${COVERAGE_DIR}
 	# gotestcover is needed to fetch coverage for multiple packages
 	go get ${COVERAGE_TOOL_REPO}
-	# testify is needed to for unit and integration tests
+	# testify is needed for unit and integration tests
 	go get ${TESTIFY_TOOL_REPO}
+	# libcompose is needed for integration tests
+	go get ${LIBCOMPOSE_TOOL_REPO}
```

Comment:
Could we add to the follow up issue to research methods on how this could be vendored or at least versioned. Because I worry that over time some builds will stop working because of an incompatible version. I was hoping `govendor get ...@1.7.2` would do something like this, but this will put it into the vendor folder instead of the global one.

Reply:
https://github.com/elastic/beats/issues/5050 feel free to update the TODO there